### ISSUE TITLE
TypeError: picker.destroy is not a function

### ISSUE THREAD

**Issue by christianbundy** [ssbc/patchwork#986]
Looks like gatherings are currently broken, I can't create a new one:

```
TypeError: picker.destroy is not a function
    at HookBinding._release (/home/christianbundy/src/ssbc/patchwork/modules/gathering/sheet/edit.js:269:25)
    at HookBinding.unbind (/home/christianbundy/src/ssbc/patchwork/node_modules/mutant/lib/apply-properties.js:165:12)
    at invokeUnbind (/home/christianbundy/src/ssbc/patchwork/node_modules/mutant/html-element.js:326:11)
    at Array.forEach (<anonymous>)
    at unbind (/home/christianbundy/src/ssbc/patchwork/node_modules/mutant/html-element.js:303:25)
    at walk (/home/christianbundy/src/ssbc/patchwork/node_modules/mutant/lib/walk.js:4:5)
    at handleChange (/home/christianbundy/src/ssbc/patchwork/node_modules/mutant/html-element.js:197:5)
    at Array.forEach (<anonymous>)
    at MutationObserver.onMutate (/home/christianbundy/src/ssbc/patchwork/node_modules/mutant/html-element.js:127:11)
```

**Comment by christianbundy** [comment]
Also getting this:

```

/home/christianbundy/src/ssbc/patchwork/node_modules/textarea-caret-position/index.js:68 GET file:///home/christianbundy/src/ssbc/patchwork/assets/null net::ERR_FILE_NOT_FOUND
(anonymous) @ /home/christianbundy/src/ssbc/patchwork/node_modules/textarea-caret-position/index.js:68
CaretCoordinates @ /home/christianbundy/src/ssbc/patchwork/node_modules/textarea-caret-position/index.js:67
module.exports @ /home/christianbundy/src/ssbc/patchwork/node_modules/suggest-box/index.js:11
(anonymous) @ /home/christianbundy/src/ssbc/patchwork/modules/suggest.js:21
bind @ /home/christianbundy/src/ssbc/patchwork/node_modules/mutant/lib/apply-properties.js:159
invokeBind @ /home/christianbundy/src/ssbc/patchwork/node_modules/mutant/html-element.js:322
rebind @ /home/christianbundy/src/ssbc/patchwork/node_modules/mutant/html-element.js:288
walk @ /home/christianbundy/src/ssbc/patchwork/node_modules/mutant/lib/walk.js:4
handleChange @ /home/christianbundy/src/ssbc/patchwork/node_modules/mutant/html-element.js:192
onMutate @ /home/christianbundy/src/ssbc/patchwork/node_modules/mutant/html-element.js:127
childList (async)
(anonymous) @ /home/christianbundy/src/ssbc/patchwork/modules/sheet/display.js:15
(anonymous) @ /home/christianbundy/src/ssbc/patchwork/node_modules/depject/apply.js:18
getCurrentValues @ /home/christianbundy/src/ssbc/patchwork/modules/gathering/sheet/edit.js:40
getCurrentValues @ /home/christianbundy/src/ssbc/patchwork/modules/gathering/sheet/edit.js:206
(anonymous) @ /home/christianbundy/src/ssbc/patchwork/modules/gathering/sheet/edit.js:26
(anonymous) @ /home/christianbundy/src/ssbc/patchwork/node_modules/depject/apply.js:18
createGathering @ /home/christianbundy/src/ssbc/patchwork/modules/page/html/render/gatherings.js:43
```

**Comment by christianbundy** [comment]
Resolved by #987.
